scripts/oncotree_to_oncotree.py

```diff
@@ -476,16 +477,17 @@ def write_summary_file(target_file, source_version, target_version):
     # For each category, codes with more granular codes introduced are shown first
     unmappable_codes = sorted([unmappable_code for unmappable_code, unmappable_node in GLOBAL_LOG_MAP.items() if unmappable_node[NEIGHBORS_FIELD]])
     ambiguous_codes = sorted([ambiguous_code for ambiguous_code, ambiguous_node in GLOBAL_LOG_MAP.items() if len(ambiguous_node[CHOICES_FIELD]) > 1], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
-    resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1 and not ("???" in resolved_node[CHOICES_FIELD]) ], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
+    partially_resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1 and not ("???" in resolved_node[CHOICES_FIELD]) and resolved_node[CLOSEST_COMMON_PARENT_FIELD]], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
```

Comment:
maybe make this line and the previous line parallel ... both have a condition that the choice field not be "???" ... but one does `not ("???" in choices)` and the other does `("???" not in choices)` ... if these mean the same thing, why not use the same construction in both lines. Also, one line has the condition earlier and one has it later .. why not put this test first in both lines?